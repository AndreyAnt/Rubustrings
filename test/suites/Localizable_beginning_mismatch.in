" Original string starting with whitespace" = "Translation without the whitespace at the very beginning";
