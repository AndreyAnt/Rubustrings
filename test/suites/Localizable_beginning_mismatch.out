[34mProcessing file: "test/suites/Localizable_beginning_mismatch.in"[0m
test/suites/Localizable_beginning_mismatch.in:1: error: beginning mismatch: " Original string starting with whitespace" = "Translation without the whitespace at the very beginning";
[31m[1m
Result: âœ˜ Some errors detected[0m[0m
